test/unit/mods.example.multitype.ts

```diff
@@ -371,6 +397,74 @@ const ValidTypeCombinations = new Map<string, ExampleSucceedingMod>(
   }),
 );
 
+const MultiTypeWithREDmodSuccesses = new Map<string, ExampleSucceedingMod>([
+  [
+    `MultiType: REDmod installable with other types except old-style archives`,
+    {
+      expectedInstallerType: InstallerType.MultiType,
+      fsMocked: mockedFsLayout(
+        {
+          [REDMOD_BASEDIR]: {
+            myRedMod: {
+              [REDMOD_INFO_FILENAME]: myRedModCompleteInfoJson,
+            },
+          },
+        },
+      ),
+      inFiles: [
+        ...CET_PREFIXES,
+        path.join(`${CET_PREFIX}/exmod/`),
+        path.join(`${CET_PREFIX}/exmod/Modules/`),
+        path.join(`${CET_PREFIX}/exmod/Modules/morelua.lua`),
+        path.join(`${CET_PREFIX}/exmod/${CET_INIT}`),
+        ...REDS_PREFIXES,
+        path.join(`${REDS_PREFIX}/rexmod/script.reds`),
+        ...TWEAK_XL_PATHS,
+        path.join(`${TWEAK_XL_PATH}\\tw\\mytweak.yaml`),
+        ...RED4EXT_PREFIXES,
+        path.join(`${RED4EXT_PREFIX}/script.dll`),
+        ...ARCHIVE_PREFIXES,
```

Comment:
probably wouldn't have this in real world. almost seems like you are passing with an oldstyle archive file.

Reply:
```suggestion
```